content/general/pubsub.textile

```diff
@@ -0,0 +1,21 @@
+---
+title: Pub/Sub
+section: general
+index: 23
+---
+
+Pub/Sub is a shorthand for the Publish/Subscribe architectural pattern, which is the most popularly used pattern in realtime data delivery. This messaging pattern lets any number of publishers publish data, ideally to a data channel/topic, which could be subscribed to by any number of subscribers. The important thing to note about Pub/Sub is that once subscribed, the subscribers no longer have to poll the server or data provider to check if there’s any new data that they need to be aware of, instead, they will be notified of it, as it becomes available.
```

Comment:
This should be "Pub/sub is shorthand.." ,the "a" is not necessary

Reply:
I think replace "which is the most popularly used pattern in realtime data delivery" with "which is a popular used pattern for realtime data delivery".  Note that popularly should have been popular anyway.

Reply:
Perhaps prefix "The important thing to note about Pub/Sub is that once subscribed..." with "The important thing to note about Pub/Sub is that publishers and subscribers are completely decoupled.  In addition, once subscribed..."